schema/message-groups.schema.json

```diff
@@ -52,12 +56,8 @@
           }
         }
       }
-    },
-    "type": {
-      "type": "string",
-      "description": "Local auto-generated group or remote group configuration from RS.",
-      "enum": ["remote-settings", "local", "default"]
```

Comment:
This field is still present, so we should update this in m-c to re-add this field 

https://searchfox.org/mozilla-central/rev/2bbb0c0a90df20702df8c8011a8996536a83cb75/browser/app/profile/firefox.js#1535-1542

Reply:
Waiting on https://bugzilla.mozilla.org/show_bug.cgi?id=1782113 to merge to m-c